src/components/trades/Trades.vue

```diff
@@ -262,7 +262,21 @@ export default class extends Mixins(PaneMixin) {
       if (amount >= this._minimumThresholdAmount * multiplier) {
         this.appendRow(trade, amount, multiplier)
       } else if (amount > this._audioThreshold) {
-        this._thresholdsAudios[0][trade.side](audioService._play, amount / (this._significantThresholdAmount * multiplier), trade.side, 0)
+        if (this._thresholdsAudios[0][trade.side].toString().includes('function anonymous(play,')) {
+          ;(this._thresholdsAudios[0][trade.side] as AudioFunction)(
+            audioService._play,

```

Comment:
I was passing audioService._play as argument because the audio script function is in strict mode... (but is it really usefull, performance wise?)
now that there is two functions (play and playurl) should we pass the whole audioService as argument ?
Easiest would be to pass both as argument ? What do you think